lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2472,8 +2477,113 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
             tx `shouldBe` Left ErrBalanceTxMaxSizeLimitExceeded
 
     describe "when passed unresolved inputs" $ do
-        xit "may fail"
-            $ property prop_balanceTransactionUnresolvedInputs
+        it "fails with some error" $ do
+            pendingWith "currently silently ignored"
```

Comment:
Note: this means that `balanceTx` currently silently ignores the condition tested in this test.